server/webdriver_tests/place_explorer_test.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import urllib
+
+from flask import abort, url_for
+from flask_testing import LiveServerTestCase
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.action_chains import ActionChains
+import logging
+import time
+
+from main import app
+
+
+USA_URL = '/place?dcid=country/USA'
+USA_DEMOGRAPHICS_URL = '/place?dcid=country/USA&topic=Demographics'
+
+class TestBase(LiveServerTestCase):
```

Comment:
can you have a new file called base_test.py which just have this base class, and all other *_test.py will inherit it.
The TestBase does not have url specific thing but just contains create_app(), setUp() and tearDown()